src/dune_rules/odoc.ml

```diff
@@ -203,20 +203,20 @@ let odoc_ext = ".odoc"
 module Mld : sig
   type t
 
-  val create : Path.Build.t -> t
+  val create : Path.Build.t * string -> t
   val odoc_file : doc_dir:Path.Build.t -> t -> Path.Build.t
   val odoc_input : t -> Path.Build.t
 end = struct
-  type t = Path.Build.t
+  type t = Path.Build.t * string
```

Comment:
I think this would be clearer as a record type so the `string` can be given a name.